examples/configFiles/deepMedic/train/trainConfig.cfg

```
@@ -61,7 +63,7 @@ typeOfSamplingForTraining = 3
 #  How many epochs to train for. Default: 35
 numberOfEpochs = 35
 #  How many subepochs comprise an epoch. Every subepoch I get Accuracy reported. Default: 20
-numberOfSubepochs = 20
+numberOfSubepochs = 1
```

This changes the default trainConfig arbitrarily. Nothing to do with pr change. Avoid such changes, they can go unnoticed.
```suggestion
numberOfSubepochs = 20
```